src/coreComponents/physicsSolvers/fluidFlow/IsothermalCompositionalMultiphaseBaseKernels.hpp

```diff
@@ -1240,28 +1415,39 @@ class SolutionCheckKernel : public ScalingAndCheckingSystemSolutionKernelBase< i
    * @param[in] compDens the component density vector
    */
   SolutionCheckKernel( integer const allowCompDensChopping,
+                       integer const localScaling,
```

Comment:
If possible can we propagate `NonlinearSolverParameters::ScalingType` just to highlight the values expected by this parameter?

Reply:
Modified, please have a look